snap/snapcraft.yaml

```diff
@@ -843,18 +853,21 @@ parts:
   app-service-config:
     plugin: nil
     stage-snaps:
-      - edgex-app-service-configurable/2.0/stable
+      - edgex-app-service-configurable/2.1/stable
     stage:
+      - -res/external-mqtt-trigger/configuration.toml
       - -res/functional-tests/configuration.toml
       - -res/http-export/configuration.toml
       - -res/mqtt-export/configuration.toml
       - -res/push-to-core/configuration.toml
       - -res/sample/configuration.toml
 
+      - -usr/*
```

Comment:
This includes the attribution and license files. It should stay included.

Reply:
Excluded.